mpas_analysis/config.default

```diff
@@ -290,83 +290,91 @@ baseDirectory = /dir/to/seaice/reference
 # directory where ocean reference simulation results are stored
 baseDirectory = /dir/to/seaice/reference
 
-[timeSeriesOHC]
+[timeSeriesOHCAnomaly]
 ## options related to plotting time series of ocean heat content (OHC)
+## anomalies from year 1
 
-# plot S, T, and OHC fields themselves, in addition to their anomalies?
-plotOriginalFields = False
-# compare to observations?
-compareWithObservations = False
```

Comment:
I see this has been removed: we were probably no longer using `compareWithObservations' anyway. At some point, though, we will want to process the OHC observations for transient/historical runs. This will only make sense for those runs, so we will need some sort of flag to turn on/off the comparison with obs.